tools/migrate-hints/test/transforms_test.js

```diff
@@ -0,0 +1,31 @@
+const test = require('ava')
+const {transformV0ToV1} = require('../lib/transforms')
+
+test('transforms v0 to v1 schema', t => {
+  const original = [{
+      general_overview: {
+        objective: "test objective",
+        "starting-point": "kubectl pod exec.",
+        "num-hints": "2"
+      }
+    }, {
+      hints: {
+        "hint-1": "test hint 1",
+        "hint-2": "test hint 2"
+      }
+    }
+  ]
+
+  const expected = [{
+      kind: "cp.simulator/scenario:1.0.0",
```

Comment:
How is this parameter delivered?

I can see in the format the reference "semver-version", but I cannot pin it back to the test.

Reply:
it's just a hardcoded string in the transform function.  I've arbirtrarily chosen v1.0.0.  This is the version of the scema and not of simulator itself.  We can use it to detect old scenarios, automate upgrading etc in the future if we wish